.codecov.yml

```
@@ -1 +1,18 @@
 comment: off
+
+flags:
+  backend:
+    paths:
+      - /pontoon/**.py
```

```suggestion
      - pontoon/**.py
```
I think this might work

---

Thanks, that seems to have done the trick. I pushed each path individually, and the backend tests already are marked as carry-forward until the ones from the current test run made it in.